[PR_Github #19089](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/19089/) Bot args parsing error: usage: /bot run [--reuse-test [optionalpipeline-id]] [--disable-fail-fast]
                [--skip-test] [--stage-list "A10-PyTorch-1, xxx"]
                [--gpu-type "A30, H100_PCIe"] [--test-backend "pytorch, cpp"]
                [--multi-gpu-test] [--add-multi-gpu-test]
                [--only-multi-gpu-test] [--disable-multi-gpu-test]
                [--post-merge]
                [--extra-stage "H100_PCIe-TensorRT-Post-Merge-1, xxx"]
                [--debug] [--detailed-log]
/bot run: error: argument --reuse-test/--disable-reuse-test: invalid int value: 'pipeline-id'

--- a/tests/integration/test_lists/waives.txt
+++ b/tests/integration/test_lists/waives.txt
@@ -346,3 +346,7 @@ accuracy/test_llm_api_pytorch.py::TestLlama4ScoutInstruct::test_fp8_chunked_pref
 accuracy/test_llm_api_pytorch.py::TestDeepSeekV3Lite::test_bfloat16_4gpus[ep4-mtp_nextn=2-attention_dp=True-cuda_graph=True-overlap_scheduler=True-torch_compile=False] SKIP (https://nvbugs/5522746)
 accuracy/test_llm_api_pytorch.py::TestDeepSeekV3Lite::test_nvfp4_4gpus[moe_backend=CUTLASS-mtp_nextn=0-tp2pp2-fp8kv=False-attention_dp=False-cuda_graph=False-overlap_scheduler=False-torch_compile=False] SKIP (https://nvbugs/5522746)
 test_e2e.py::test_ptp_quickstart_multimodal[NVILA-8B-FP16-vila/NVILA-8B-image-False] SKIP (https://nvbugs/5523925)
+test_e2e.py::test_ptp_quickstart_multimodal[NVILA-8B-FP16-vila/NVILA-8B-image-False] SKIP (https://nvbugs/5509024)
+test_e2e.py::test_ptp_quickstart_multimodal[qwen2.5-vl-7b-instruct-Qwen2.5-VL-7B-Instruct-video-False] SKIP (https://nvbugs/5509024)
+test_e2e.py::test_ptp_quickstart_multimodal[qwen2.5-vl-7b-instruct-Qwen2.5-VL-7B-Instruct-video-True] SKIP (https://nvbugs/5509024)
+test_e2e.py::test_trtllm_multimodal_benchmark_serving SKIP (https://nvbugs/5523315)

